src/components/Onboarding/S1GetStarted.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import onboardingPic from '../../assets/onboardingPic.png';
+
+function S1GetStarted() {
+	return (
+		<div>
+			<h1>Getting Started</h1>
+			<h5>You're only a few steps away from getting started on Kansha</h5>
+			<div className="list-container">
```

Comment:
please refactor this - instead of using `div` use `ol` or `ul`, and instead of `p` tags, use `li`

Reply:
we fixed this!